micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,98 @@
+      return localModuleName.replace(/[^a-zA-Z0-9]/g, "-").toUpperCase();
+  };
+  const localizationModule = tranformLocModuleName(`DIGIT-MDMS-${rawSchemaCode}`).toLowerCase();
+
+  const createLocalizationCode = (fieldName, fieldValue) => {
+    const upperFieldName = fieldName.toUpperCase();
+    const transformedValue = (fieldValue || "").replace(/\s+/g, "").toUpperCase();
+    return tranformLocModuleName(`${rawSchemaCode}_${upperFieldName}_${transformedValue}`);
+  };
+
+  let localizationCodes = [];
+  if (data && data.data && localisableFields.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)